### PR TITLE
Components tests

### PR DESCRIPTION
**Pull request by pink-bunny**
Component test coverage except:
- components/AppRoot;
- pages/NotFound.

### CODE REVIEW

File: src/pages/MyLists/__tests__/component.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import MyLists from '../component';
+
+test('MyLists component matches snapshot', () => {
```
Comment (plzen) [inline src/pages/MyLists/__tests__/component.test.js]: Wrap all in `describe` block and use `it` for test

**Comment by plzen** [comment]
Please, add test for pages/NotFound as we discussed on daily meeting

File: src/components/Button/__tests__/component.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Button from '../index';
+
+describe('Button component matches snapshot', () => {
```
[the remark this answers is not in the document]
Comment (pink-bunny) [reply src/components/Button/__tests__/component.test.js]: Does it relate to other tests description or only to the button?

File: src/components/Button/__tests__/component.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  it('without props', () => {
```
[the remark this answers is not in the document]
Comment (pink-bunny) [reply src/components/Button/__tests__/component.test.js]: Done

File: src/components/Button/__tests__/component.test.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    const wrapper = shallow(<Button />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('with custom props', () => {
```
[the remark this answers is not in the document]
Comment (pink-bunny) [reply src/components/Button/__tests__/component.test.js]: Done

File: src/components/Button/__tests__/component.test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (plzen) [reply src/components/Button/__tests__/component.test.js]: Yes, for all tests

File: src/pages/Dashboard/Search/__tests__/component.test.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Search from '../component';
+
+test('Search component matches snapshot', () => {
```
[the remark this answers is not in the document]
Comment (pink-bunny) [reply src/pages/Dashboard/Search/__tests__/component.test.js]: Done

File: src/pages/Dashboard/__tests__/component.test.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  it('with required props', () => {
+    const wrapper = shallow(<Dashboard {...requiredProps} />);/* eslint-disable-line */
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('with custom props', () => {
+    const props = {
+      ...requiredProps,
+      list: []
```
[the remark this answers is not in the document]
Comment (pink-bunny) [reply src/pages/Dashboard/__tests__/component.test.js]: Done

File: src/components/Button/__tests__/component.test.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    const props = {
+      text: 'Custom button',
+      type: 'dashed',
+      htmlType: 'submit',
+      loading: true,
+      onClick: jest.fn()
+    };
+    const wrapper = shallow(<Button {...props} />); /* eslint-disable-line */
```
[the remark this answers is not in the document]
Comment (pink-bunny) [reply src/components/Button/__tests__/component.test.js]: Would be done in the next tests

File: src/pages/MyLists/__tests__/component.test.js (hunk 1 above)
Comment (pink-bunny) [reply src/pages/MyLists/__tests__/component.test.js]: Done

**Pull request merged**